### ISSUE TITLE
Missing vuex dependency breaks builds in quasar/app@3.0.0-rc.1

### ISSUE THREAD

**Issue by akshay-sood** [quasarframework/quasar#9616]
**Describe the bug**
In quasar/app@3.0.0-rc.1 package.json, vuex is not included. This breaks the compilation with error: 
 ```
App •  ERROR  •  UI  in ./src/store/index.js

Module not found: Can't resolve imported dependency "vuex"
Did you forget to install it? You can run: npm install --save vuex
```

**Codepen/jsFiddle/Codesandbox (required)**
I don't know how to simulate a npm package.json issue on Codepen. Sorry about that.

**To Reproduce**

1. Upgrade Quasar/app to latest release candidate - I upgraded from 3.0.0-beta.1
2. Npm install to reinstall all dependencies
3. Run quasar dev

**Expected behavior**
The app compiles for dev and runs.

**Screenshots**
Error:
![image](https://user-images.githubusercontent.com/50393320/121303761-16ded600-c919-11eb-9488-fd307a99d3e3.png)

My package.json
![image](https://user-images.githubusercontent.com/50393320/121303859-3249e100-c919-11eb-8825-afbbd031fd25.png)

quasar/app rc package.json
![image](https://user-images.githubusercontent.com/50393320/121303940-4db4ec00-c919-11eb-96da-d2bfd754ff04.png)

**Platform (please complete the following information):**
Quasar Version: 2.0.0-rc.1
@quasar/app Version: 3.0.0-rc.1
Quasar mode:
  - [x] SPA
  - [ ] SSR
  - [ ] PWA
  - [ ] Electron
  - [ ] Cordova
  - [ ] Capacitor
  - [ ] BEX

Tested on:
  - [x] SPA
  - [ ] SSR
  - [ ] PWA
  - [ ] Electron
  - [ ] Cordova
  - [ ] Capacitor
  - [ ] BEX

OS: Windows 10
Node: v14.17
NPM: 6.14.13
Yarn:
Browsers: N.A.
iOS: N.A.
Android: N.A.
Electron: N.A.

**Additional context**

**Comment by akshay-sood** [comment]
I can create a PR for this if you need me to?

**Comment by heartbeatLV** [comment]
in the [v3.0.0-rc.1](https://github.com/quasarframework/quasar/releases/tag/%40quasar%2Fapp-v3.0.0-rc.1) release a breaking change has been introduced:

> Not supplying "vuex" package anymore. If using a store, please yarn add vuex@4 / npm install vuex@4.
> This makes sense because decoupling Vuex will allow developers to upgrade to the future Vuex 5 without us making any breaking changes.

**Comment by akshay-sood** [comment]
Wow, apologies, my bad! Thanks for the quick turnaround though! 

Would someone mind if I suggest an edit on the docs (next.quasar.dev) to mention this as well?

**Comment by rstoenescu** [comment]
@akshay-sood already did. docs are being redeployed

**Comment by githrdw** [comment]
@rstoenescu can it be that `vuex` has also disappeared from the ssr-generated `package.json`?

As you can see left is my root `package.json` where vuex is installed, but after running `quasar build -m ssr` vuex is not available in the `./dist/ssr/package.json`.

Am I overseeing something? I also tried to create a blank new project and this problem occured as well.
My build is running in an automated pipeline, so installing vuex seperately from the rest is a little awkward.

![image](https://user-images.githubusercontent.com/1455140/122294542-72502b80-cef8-11eb-905a-a2f7caa8d923.png)


```
 Build mode........ ssr
 Pkg quasar........ v2.0.0-rc.3
 Pkg @quasar/app... v3.0.0-rc.3
 Pkg webpack....... v5
 Debugging......... no
 Publishing........ no
 Transpiled JS..... yes (Babel)
 ```

**Comment by githrdw** [comment]
I think the following line tries to set `vuex` to a version that does not exists in the `cliPkg.dependencies`?

https://github.com/quasarframework/quasar/blob/56d4504a78daf576249e58cfe5b5ebfc9a6e725a/app/lib/webpack/ssr/plugin.webserver-assets.js#L63

**Comment by rstoenescu** [comment]
Fix will be available in "@quasar/app" v3.0.0 (stable)
